scripts/create_elmo_embeddings_from_vocab.py

```diff
@@ -0,0 +1,87 @@
+# pylint: disable=no-self-use
+
+import os
+import argparse
+import sys
+
+sys.path.insert(0, os.path.dirname(os.path.abspath(os.path.join(__file__, os.pardir))))
```

Comment:
Why is this necessary?

Reply:
Ah it just means you can run the script from the top level directory and import things as though this script resided in the top level directory.